app/MMB/src/components/exploreCity/CardContent.js

```diff
@@ -13,17 +16,45 @@ type Props = {|
   +data: BookingType,
 |};
 
-const CardContent = (props: Props) => {
-  const type = idx(props.data, _ => _.__typename);
-  return (
-    <React.Fragment>
-      {type === 'BookingOneWay' && <ExploreOneWay data={props.data} />}
-      {type === 'BookingReturn' && <ExploreReturn data={props.data} />}
-      {type === 'BookingMulticity' && <ExploreMulticity data={props.data} />}
-      {/* TODO: Add button */}
-    </React.Fragment>
-  );
-};
+class CardContent extends React.Component<Props> {
+  onPress = () => {
+    console.warn('TODO:'); // This button should take you directly to the city shown by explore text component
+  };
+
+  render = () => {
+    const type = idx(this.props.data, _ => _.__typename);
+    return (
+      <React.Fragment>
+        {type === 'BookingOneWay' && <ExploreOneWay data={this.props.data} />}
+        {type === 'BookingReturn' && <ExploreReturn data={this.props.data} />}
+        {type === 'BookingMulticity' && (
+          <ExploreMulticity data={this.props.data} />
+        )}
+        <View style={styles.row}>
+          <View>
+            <TextButton
+              title={
+                <Translation id="mmb.main_menu.explore_city.card_content.explore" />
```

Comment:
I believe this title should sometimes display the name of the city to explore too, e.g. Prague-Copenhagen shows `Explore Copenhagen`.

Reply:
According to zeplin slides yes, I the real application it does not. 

Zeplin is normally outdated 😞

Reply:
![img_5562](https://user-images.githubusercontent.com/7705041/43709567-79493864-996d-11e8-97e6-137db06e254d.PNG)

Reply:
Oh really? I just checked on the native app on the iPhone, and I could see Explore Copenhagen, so I am not sure.

Reply:
🙈 the above screenshot is actually from the real application also. 

It seems that for a multicity trip in progress, it will show only `Explore`

for an upcoming trip it will show `Explore <city>` 

I will treat this as a 🐛 from the native developers, and always do `Explore <city>` I believe it should not be different.

Reply:
I agree with this decision. I will wait for you to update the PR to approve it then.